README.md

```diff
@@ -95,6 +95,7 @@
 | ⭐ [Linux Kernel Mentorship Program](https://wiki.linuxfoundation.org/lkmp)            | Yes  | [Mar - May](https://wiki.linuxfoundation.org/lkmp/lkmp_schedule)  | mid-Jan             | late-Feb   | Mentorship Program  |  Certificate  |
 | ⭐ [Hyperledger Mentorship Program](https://wiki.hyperledger.org/display/INTERN/How+to+Apply)             | Yes  | [Jun - Aug(Full-Time) Jun - Nov(Part-time)](https://wiki.hyperledger.org/display/INTERN#HyperledgerMentorshipProgram-SummerFT(Full-Time)MenteeSchedule)  | Jan-28                | Mar-10       | Mentorship Program   | Certificate  |
 | ⭐ [Outreachy](https://www.gnome.org/outreachy/)                                  | Yes  | [May - Aug](https://www.outreachy.org/apply/project-selection/)  | late Jan            | late Feb | Internship  | Certificate  |
+| [Summer of Bitcoin](https://www.summerofbitcoin.org/) | Yes (3000 USD) | [Jan - Aug](https://www.summerofbitcoin.org/how-it-works) | January 1 | March 15 | Summer Internship Program | Internship, Job opportunity (if performed exceptionally well)
```

Comment:
looking Perfect .

Reply:
@saintmalik  thanks for the suggestion, will keep this in mind😊